### ISSUE TITLE
Proxy setting?

### ISSUE THREAD

**Issue by unsystemizer** [Jpja/CounterTools#14]
Would it be possible to accommodate a proxy setting (SOCKS5?)

**Comment by Jpja** [comment]
I'm not familiar with  SOCKS but you can use CounterTools behind a proxy, e.g. with TOR, granted the exit node is not blacklisted by both the primary and backup APIs. 
If this is an issue, it will be trivial to add more API backups.

**Comment by unsystemizer** [comment]
Yeah, I was thinking about making it more flexible (so that both Tor and any other VPN proxy can be used), but that's also a possibility.
I tried to open CounterTools from a browser which uses Tor as its SOCKS5 proxy, also from Tor Browser. If setup correctly, Tor SOCKS5 proxy seems to work (i.e. it does work but I didn't carefully check if there are any leaks; DNS and other). 

With Tor Browser I couldn't get non-BTC balances to show (USD exchange rate), maybe due to pure unluck - some sites block some exit hosts so who knows. I tried to change to a different relay but it didn't help. 

The last point is there are spoof sites so there may be some risk in using Tor exit nodes.

I guess we can close this - the solution is to set SOCKS5 in your browser, then open CT in that browser.
